qt/Session.h

```diff
@@ -24,6 +25,7 @@
 #include "RpcQueue.h"
 #include "Torrent.h"
 #include "Typedefs.h"
+#include "Utils.h" // std::hash<QString>
```

Comment:
`std::map<K, V>` doesn't need `std::hash<K>`.

Reply:
Fixed in 73bf1f7